src/frontend/src/components/common/Header/Header.tsx

```diff
@@ -140,6 +140,7 @@ function Header({ user }: Props) {
           cancelLabel="Cancel"
           titleText="Sign out confirmation"
           show={showModal}
+          reversed={true}
```

Comment:
From my understanding only Header calls modal and it seems to be called once. Can we default the reversed to be true? 
If not the case, ignore this message. 
I see it being called again in PIAIntakeForm

Reply:
The modal is called now for every CTA on the PIA intake. The sign out modal is the only one where the buttons are in reverse order so I consider that to be the 'reverse' or opposite of the other implementations. In order, to keep this modal the same I've added the optional property of 'reverse'.